src/randoop/main/GenInputsAbstract.java

```diff
@@ -408,16 +366,6 @@ public GenInputsAbstract(String command, String pitch,
   @Option("Run Randoop but do not create JUnit tests")
   public static boolean dont_output_tests = false;
   
-  /**
-   * Output sequences even if they do not complete execution.
-   * 
-   *  Randoop's default behavior is to output only tests consisting of
-   *  method call sequences that execute every statement, rather than throwing
-   *  an exception or failing a contract check before the last statement.
-   */
-  @Option("Output sequences even if they do not complete execution")
-  public static boolean output_nonexec = false;
-  
   @Option("specifies regex of classes that must be in any regression tests")
```

Comment:
I know this isn't your fault, but I don't find this documentation descriptive.  Also, please capitalize the first letter of every `@Option` argument, for consistency.